### PR TITLE
ADEN-9627 Upgrade Prebid.js and enable CCPA module

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: webpack.dev.js [hunk 1]
```diff
@@ -1,5 +1,4 @@
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
-const CopyWebpackPlugin = require('copy-webpack-plugin');
```
Comment (fraszczakszymon) [inline webpack.dev.js]: It seems that we no longer need this dependency in `package.json`

File: src/ad-engine/services/pbjs-factory.ts [hunk 2]
```diff
@@ -13,17 +15,14 @@ class PbjsFactory {
...
 
-			if (libraryUrl) {
-				scriptLoader.loadScript(libraryUrl, 'text/javascript', true, 'first');
-			} else {
-				logger(
-					logGroup,
-					'Prebid library URL not defined. Assuming that window.pbjs will be loaded.',
-				);
-			}
+			scriptLoader.loadScript(libraryUrl || prebidLibraryUrl, 'text/javascript', true, 'first');
```
Comment (fraszczakszymon) [inline src/ad-engine/services/pbjs-factory.ts]: What about `icPrebidVersion`?

File: webpack.dev.js (hunk 1 above)
Comment (dmnsobczak) [reply webpack.dev.js]: You're right! Removed.

File: src/ad-engine/services/pbjs-factory.ts (hunk 2 above)
Comment (dmnsobczak) [reply src/ad-engine/services/pbjs-factory.ts]: It will be added under `libraryUrl` in application.

**Pull request merged**